apps/wallet-mobile/src/WalletInit/WalletInitNavigator.tsx

```diff
@@ -43,15 +47,21 @@ export const WalletInitNavigator = () => {
       />
 
       <Stack.Screen
-        name="create-wallet-form"
-        component={CreateWalletScreen}
-        options={{title: strings.createWalletTitle}}
+        name="wallet-details-form"
+        component={WalletDetailsScreen}
+        options={{...DEPRECATED_defaultStackNavigationOptions, title: strings.createWalletTitle}}
```

Comment:
this shouldn't be used. this was fixed:

`...defaultStackNavigationOptions(theme)`